SDWebImage/SDWebImageWebPCoder.m

```diff
@@ -343,10 +352,12 @@ - (nonnull CGColorSpaceRef)sd_colorSpaceWithDemuxer:(nonnull WebPDemuxer *)demux
     
     int result = WebPDemuxGetChunk(demuxer, "ICCP", 1, &chunk_iter);
     if (result) {
-        NSData *profileData = [NSData dataWithBytes:chunk_iter.chunk.bytes length:chunk_iter.chunk.size];
+        NSData *profileData = [NSData dataWithBytesNoCopy:(void *)chunk_iter.chunk.bytes length:chunk_iter.chunk.size freeWhenDone:NO];
```

Comment:
This seems good. CGColorSpace will copy the ICC profile data. So this don't need a extra copy...